README.md

```diff
@@ -25,9 +63,645 @@ What's been tested: server edition on azure, desktop edition in virtualbox VM, d
+| 11 | `<SMTP Server Port>`                 | SMTP Server Port                                                                                                                                                                                                            |
+| 12 | `<SMTP Auth User> `                  | SMTP Auth user where the email will be sent fromSMTP Server Name                                                                                                                                                            |
+| 13 | `<SMTP Auth User password> `         | SMTP Auth user credentials. Note: You can test the SMTP credentials by writing sample SMTP application before starting the deployment.                                                                                      |
+| 14 | AzureCliVersion                    | You can get Azure CLI version by running “az --version” in the windows command prompt. If you follow this guide, the AzureCliVersion will be 2 as shown in the example deployment script.                                   |
+| 15 | PlatformName                       | Name of the site. This will appear in several places in the site: Currently defaulted to Contoso Learning. Ex: Change to “Contoso Academy”                                                                                  |
+| 16 | PlatformEmailAddress               | Email address used by the platform (application) as default email address for sending email messages                                                                                                                        |
+
+##### 3.1.1. Enlistment Root
+This is the root of when you sync’d the files from GitHub. See step Sync Configuration Files.
+##### 3.1.2. Cluster Name
+Choose a unique name for your Open edX on Azure cluster. After deployment, this will be the name
+shown in the Azure Portal under Resource groups. All the resources will be provisioned using this prefix
+in your resource group.
+
+Tip: Limit the Cluster Name to 8 lowercase alpha-numeric characters. Numbers are fine, but avoid %, $,
+etc.
+##### 3.1.3. Location
+This is the Azure location where the VMs will be deployed. Choose the closest Azure data center location
+to your geographical area. An example is “east us”.
+##### 3.1.4. Located the correct AAD
+Most likely you already have an Active Directory associated with your organization or subscription. Go to
+http://portal.azure.com, select your login name on the top righthand side of the screen. The Azure Active
+Directories associated with your login are displayed. Select the default directory associated with your
+account, or if you logged into Azure with your work or school account, this would likely be name of your
+organization. 
+
+![locate_aad](images/figure-3.png "Locate your Azure Active Directory")
```

Comment:
Suggest: Should replace image with one that doesn't contain an employee's name

Reply:
I picked this from pdf documentation. In that case we need to update that as well